packages/nuxeo-platform-explorer-package/src/main/resources/install/templates/explorer-deploy/nuxeo.prod

```diff
@@ -0,0 +1,153 @@
+default.target=.
+
+## all templates to apply on target deployment env
+nuxeo.template.includes=default,s3binaries,postgresql,explorer-sitemode,explorer-virtualadmin
+# TODO: check logs config with 'docker' or 'docker-json' template
+
+## explorer confs mapping
+
+org.nuxeo.apidoc.apidocAdmin.password=${env:ORG_NUXEO_APIDOC_APIDOCADMIN_PASSWORD}
+
+
+## default confs mapping
+
+nuxeo.data.dir=${env:NUXEO_DATA_DIR:nxserver/data}
+nuxeo.server.dir=${env:NUXEO_SERVER_DIR:nxserver}
+
+# Default IP address binding
+nuxeo.bind.address=${env:NUXEO_BIND_ADDRESS:0.0.0.0}
+
+# publicly visible URL
+nuxeo.url=${env:NUXEO_URL:http://localhost:8080/nuxeo}
+
+# Application context path
+org.nuxeo.ecm.contextPath=${env:ORG_NUXEO_ECM_CONTEXTPATH:/nuxeo}
+
+org.nuxeo.ecm.product.name=Nuxeo Explorer
+
+nuxeo.notification.eMailSubjectPrefix=${env:NUXEO_NOTIFICATION_EMAILSUBJECTPREFIX:[Explorer] }
```

Comment:
Probably not needed to have this configurable at the docker level: you should be able to hard code this in your package if you think email will be used.

Reply:
we're not actually at the docker level here, we're in a template shipped with the package, so this is where i chose to hardcode this value (and still make it configurable via env variable if needed)